xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStream.cpp

```diff
@@ -41,7 +34,14 @@ bool CDVDInputStream::Open(const char* strFile, const std::string &content, bool
   // but keep them in m_url.
   url.SetProtocolOptions("");
   m_strFileName = url.Get();
+}
 
+CDVDInputStream::~CDVDInputStream()
+{
+}
+
+bool CDVDInputStream::Open(const std::string &content, bool contentLookup)
```

Comment:
an alternative would be to pass CFileItem and drop m_filename from video player. CFileItem holds all info required and this is passed to VideoPlayer in CVideoPlayer::OpenFile. It is silly to split this into separate parameters, isn't it. This way we wouldn't even break rtmp